app/views/comments/_comment.html.erb

```diff
@@ -0,0 +1,7 @@
+<div class="comment-card">
+  <p class="bold"><%= comment.user.name %>: <%= markdown(comment.body) %> <%= comment.created_at %><p>
+  <% if current_user == comment.user %>
+    <%= link_to "Edit Comment", edit_project_story_comment_path(project.id, story.id, comment.id), class: "link-blue" %> |
+    <%= link_to "Delete", project_story_comment_path(project.id, story.id, comment.id), method: :delete, data: { confirm: "Are you sure?", comment_id: comment.id }, remote: true , title: "Delete" %>
```

Comment:
```suggestion
    <%= link_to "Delete", project_story_comment_path(project.id, story.id, comment.id), method: :delete, data: { confirm: "Are you sure?", comment_id: comment.id }, title: "Delete" %>
```

we are not responding with a JS view and we are redirecting back, we don't need this to be an ajax request

Reply:
What, I thought I removed this before 🤔